src/wallet.ts

```diff
@@ -0,0 +1,218 @@
+import ethTx = require('ethereumjs-tx');
+import ethWallet = require('ethereumjs-wallet');
+
+declare const Buffer;
+declare const setTimeout;
+
+export default class Wallet {
+  length: number;
+  nonce: number;
+  web3: any;
+
+  constructor(web3: any) {
+    this.length = 0;
+    this.nonce = 0;
+    this.web3 = web3;
+  }
+
+  _findSafeIndex(pointer = 0) {
+    pointer = pointer;
+    if (this.hasOwnProperty(pointer)) {
+      return this._findSafeIndex(pointer + 1);
+    } else {
+      return pointer;
+    }
+  }
+
+  _currentIndexes() {
+    const keys = Object.keys(this);
+    const indexes = keys
+      .map((key) => parseInt(key, 10))
+      .filter((n) => n < 9e20)
+      .slice(0, this.length);
+    return indexes;
+  }
+
+  create(numAccounts) {
+    for (let i = 0; i < numAccounts; i++) {
+      const wallet = ethWallet.generate();
+      this.add(wallet);
+    }
+    return this;
```

Comment:
Wouldn't it be more valuable to return a bool value depending on if the function succeeded or not?

Reply:
I think this is okay. Since we don't have any failure cases this high up, we rely on the lower level code to throw errors when needed anyway. Returning the wallet object means we have it directly usable... but we should have already have it usable anyway. Should be okay to change to booleans